### ISSUE TITLE
Unix_File_Move_To_Same_Directory_Triggers_Event failed in CI on OS X

### ISSUE THREAD

**Issue by stephentoub** [dotnet/corefx#4831]
http://dotnet-ci.cloudapp.net/job/dotnet_corefx/job/osx_debug_tst_prtest/21/console
```
09:19:39    MoveFileTests.Unix_File_Move_To_Same_Directory_Triggers_Event(eventType: Changed, moveRaisesEvent: False) [FAIL]
09:19:39       Should not observe a Changed event within 500ms
09:19:39       Expected: False
09:19:39       Actual:   True
09:19:39       Stack Trace:
09:19:39             at Utility.ExpectNoEvent(WaitHandle eventOccurred, String eventName, Int32 timeout)
09:19:39             at MoveFileTests.MoveAndCheck_SameDirectory(WatcherChangeTypes eventType, Boolean moveRaisesEvent)
09:19:39             at MoveFileTests.Unix_File_Move_To_Same_Directory_Triggers_Event(WatcherChangeTypes eventType, Boolean moveRaisesEvent)
```

**Comment by stephentoub** [comment]
Also failed here:
http://dotnet-ci.cloudapp.net/job/dotnet_corefx/job/osx_release_tst_prtest/22/console

**Comment by sokket** [comment]
Will be fixed by #4781

**Comment by sokket** [comment]
Fixed
